sdk/schemaregistry/schema-registry/src/models.ts

```diff
@@ -18,6 +10,12 @@ export interface SchemaId {
   /** ID that uniquely identifies a schema in the registry namespace. */
   id: string;
 
+  /**
+   * Serialization type of schema.
+   * Currently only 'avro' is supported, but this is subject to change.
```

Comment:
I wonder if it would be useful to provide a KnownSerializationTypes enum once this list gets bigger